shared/chat/conversation/info-panel/notifications/container.js

```diff
@@ -34,28 +67,15 @@ const mapDispatchToProps = (dispatch: Dispatch): DispatchProps => ({
     dispatch(ChatGen.createToggleChannelWideNotifications({conversationIDKey})),
 })
 
-type OwnProps = {
-  channelWide: boolean,
-  conversationIDKey: string,
-  desktop: Types.NotifyType,
-  mobile: Types.NotifyType,
-  muted: boolean,
-  saveState: Types.NotificationSaveState,
-}
-
-type _Props = Props & {
-  _resetSaveState: () => void,
-}
-
-const mergeProps = (_, dispatchProps: DispatchProps, ownProps: OwnProps): _Props => {
```

Comment:
yes, if the parent is doing everything then this doesn't make sense. In this model the parent is just deciding if it should render anything at all or not, nothing more